wireguard-go-rust-wrapper/build.rs

```diff
@@ -26,12 +26,25 @@ fn main() {
 
     println!("cargo:rustc-link-lib=static=wireguard-go");
 
+    // At this stage, target_arch will be used only in windows-native build 
+    let target_arch = {
+        match env::var("CARGO_CFG_TARGET_ARCH").unwrap().as_str() {
+            "x86" => "386",
+            "x86_64" => "amd64",
+            "aarch64" => "arm64",
+            "arm" => "arm",
```

Comment:
Don't we support only 64 bit architectures?

Reply:
Hmm, but it looks like this variable is used only for the powershell script, with which only "x86_64" and "aarch64" can work, so I see no reason to consider the rest of them here. Also it look that you just use them in the ifs inside the script, so maybe you can just push the value of `env::var("CARGO_CFG_TARGET_ARCH")` to it and modify the script, so it would use `x86_64` instead of `amd64` etc.?

Reply:
This mapping exists for golang. I can remove the architectures that we are not currently build, but mapping still needs to be there, because `GOARCH` differs from `CARGO_CFG_TARGET_ARCH`.